mappings/net/minecraft/client/render/entity/ModelledEntityRenderLayer.mapping

```diff
@@ -0,0 +1,4 @@
+CLASS dmn net/minecraft/client/render/entity/ModelledEntityRenderLayer
```

Comment:
Modelled? Not Modeled?

Reply:
modelled is correct I believe

Reply:
In BrE, yes, but I think our mappings *generally* assume AmE?

Reply:
yup. It should be Modeled

Reply:
Resolved.